runtime/js/40_process.js

```diff
@@ -317,10 +316,7 @@ class ChildProcess {
   }
 
   kill(signo = "SIGTERM") {
-    if (this.#rid === null) {
-      throw new TypeError("Child process has already terminated.");
-    }
-    ops.op_kill(this.#pid, signo, "Deno.Child.kill()");
+    ops.op_spawn_kill(this.#rid, signo);
```

Comment:
@nayeemrmn could you add a check here that this functions throws once the process finishes running? Ie. make it impossible to call `kill()` on a process that did `new Command().output()`. Also please add a test for this scenario for all 3 APIs used to spawn processes.

Reply:
This method is only applicable to one of the ways `new Command().spawn()`, otherwise done.